roof_classify/roof_classify.py

```diff
@@ -189,6 +189,20 @@
 ]
 
 
+"""
+For more information on the implementation of random forests and SVM classifiers,
+see documentation below:
+* http://scikit-learn.org/dev/modules/generated/sklearn.ensemble.RandomForestClassifier.html
+* http://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html
+"""
+CLASSIFIERS = {
+    "random-forest": RandomForestClassifier(
+        n_jobs=4, n_estimators=10, class_weight="balanced"
+    ),
+    "svm": SVC(class_weight="balanced"),
+}
```

Comment:
I'm not a fan about running code in global outside a module logic because it will be run everytime QGIS starts instead of be ran only when the plugin is loaded.

Reply:
You've got a point here, and all things considered, an SVM classifier is created but never used, so I decided to drop the `CLASSIFIERS` variable and instanciate a `RandomForestClassifier` directly in the `run(.)` function (see commit https://github.com/DINFO-UniFI/RoofClassify/pull/21/commits/2529e4970827b4fabaa3d6b8c7008c0cd92dc53d)